src/gui/sdlmain.cpp

```diff
@@ -1395,11 +1379,16 @@ static GLuint BuildShader ( GLenum type, const char *shaderSrc ) {
 	return shader;
 }
 
-static bool LoadGLShaders(const char *src, GLuint *vertex, GLuint *fragment) {
-	GLuint s = BuildShader(GL_VERTEX_SHADER, src);
+static bool LoadGLShaders(const std::string_view source_sv, GLuint *vertex,
+                          GLuint *fragment)
+{
+	if (source_sv.empty() || !vertex || !fragment)
+		return false;
```

Comment:
LoadGLShaders's arguments are programmatically controlled and only called if they're valid; so this can become an `assert`.